config/data_sources/variables.tf

```diff
@@ -0,0 +1,13 @@
+# Covid Tracking Project vars
```

Comment:
are these comments accurate? They're suggesting that these are CTP-specific but the file name and variable names don't say anything about CTP

Reply:
These variables are currently all specific to CTP, but if any other datasource needs to define variables, they would do it in this file. I added a comment to help clarify.